Install-Guide/Overview.md

```diff
@@ -19,7 +19,8 @@ you read the [Gluster Admin Guide](../Administrator Guide/) to learn how to admi
 how to select a volume type that fits your needs. Also, be sure to
 enlist the help of the Gluster community via the IRC channel or Q&A
 section . We want you to be successful in as short a time as possible.
-Overview:
+
+### Overview:
```

Comment:
nit: Now that you are fixing this, could you please remove the `:` at the end as it now becomes a heading.